src/shelter/shelter.service.ts

```diff
@@ -142,6 +144,27 @@ export class ShelterService {
     );
   }
 
+  normalizeString(str: string) {
+    return str.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
+  }
+
+  async getUnaccentShelterIds(searchText: string) {
+    try {
+      if (!searchText) return [];
+      const normalizedSearch = this.normalizeString(searchText);
+
+      const parameterizedSearch = `%${normalizedSearch}%`;
+
+      const idsFound = await this.prismaService.$queryRaw<{ id: string }[]>(
+        Prisma.sql`SELECT id FROM shelters WHERE unaccent(name) ILIKE ${parameterizedSearch} OR unaccent(address) ILIKE ${parameterizedSearch};`,
+      );
+      return idsFound.map(({ id }) => id);
+    } catch (e) {
```

Comment:
Every endpoint already has a try/catch. There is no need to track (log) this information on the service.
Could you remove the try/catch in this method?